<launch>
<node pkg="traj_server" type="traj_server_node" name="real_traj_server" output="screen"
    launch-prefix="">
    <!-- receive the polynomial trajectory -->
    <remap from="~trajectory" to="/trajectory" />
    <remap from="~trajectory_arm" to="/trajectory_arm" />

    <!-- for px4 and hi-arm control -->
    <remap from="~position_cmd" to="/position_cmd" />
    <remap from="~position_cmd_delta" to="/position_cmd_delta" />
    <remap from="~/odometry" to="/ekf/ekf_odom" />

    <!-- for Delta-Display -->
    <remap from="~body/end_effector" to="/body/end_effector" />
    <remap from="~position_des" to="/position_des" />

    <param name="is_sim" value="false" />
    <param name="for_arm" value="true" />
    <param name="for_base" value="true" />
    <param name="debug" value="false" />
    <param name="init_arm_z" value="0.06" />

    <!-- If is_sim, the following parameters will be used. -->
    <param name="start_x" value="$(arg start_x)" />
    <param name="start_y" value="$(arg start_y)" />
    <param name="start_z" value="$(arg start_z)" />
  </node>
</launch>